src/Util/File.cc

```diff
@@ -34,6 +34,11 @@ bool IsCompressedFits(const std::string& filename) {
     return false;
 }
 
+bool IsGzFile(uint32_t magic_number) {
```

Comment:
This is a confusing name for a function which takes a magic number as a parameter (not a file name or file object). I would suggest `IsGzMagicNumber` for more clarity.